integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -530,3 +530,19 @@ def get_side_by_side_language_options(
                     },
                 )
         return side_by_side_language_options
+
+    def get_context_data(self, **kwargs: Any) -> dict[str, Any]:
+        context = super().get_context_data(**kwargs)
+        region = self.request.region
+        page_instance = region.pages.filter(id=kwargs.get("page_id")).first()
```

Comment:
Nit: I think it's better use .get() or even better use `get_object_or_404` instead of filter().first() :)

Reply:
Thanks, yes, I think it does make sense to raise 404 if no page is found here. Could you just quickly confirm the most recent changes cover what you meant? Then I'll squash the commits and merge

Reply:
Oh, I now remember what the problem with that was (thanks tests for failing rightfully). :sweat_smile: 
When creating a new page, this will raise an exception, as the page is not yet in the database. We could do .get and then handle the DoesNotExist exception, but in this case I'd actually prefer sticking to .filter(...).first(). Is that okay for you @JoeyStk

Reply:
Yes, I suppose that's fine :)